examples/src/components/VisualizationGeoPushpinChartByUriExample.jsx

```diff
@@ -0,0 +1,27 @@
+// (C) 2007-2020 GoodData Corporation
+import React, { Component } from "react";
+import "@gooddata/react-components/styles/css/main.css";
+import { Visualization } from "@gooddata/react-components";
+
+import { projectId } from "../utils/fixtures";
+import { MAPBOX_TOKEN, geoPushpinChartVisualizationUri } from "../utils/fixturesGeoChart";
+
+export class VisualizationTable extends Component {
```

Comment:
`VisualizationTable `?

Reply:
Not sure for this, but I saw all other part of Visualization use this name. Should I change ?

Reply:
change to VisualizationGeoPushpinChartByUri